ci/pipeline.yml

```diff
@@ -1,212 +1,356 @@
+      - task: write-cert
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: alpine
+          params:
+            CERT: ((load_balancer_cert))
+            KEY: ((load_balancer_key))
+          run:
+            path: sh
+            args: ["-c", "echo -e \"${CERT}\" > lb_cert/cert.pem && echo -e \"${KEY}\" > lb_key/key.pem"]
+          outputs:
+            - name: lb_cert
+            - name: lb_key
+      # Deploys the BOSH director via `bbl up`
+      - put: bbl-state
+        params:
+          command: up
+          name: ((bbl_env_name))
+          args:
+            lb-cert: /tmp/build/put/lb_cert/cert.pem
+            lb-key: /tmp/build/put/lb_key/key.pem
+      - get: stackdriver-tools-source
+        attempts: 2
+      # Update runtime config to add BOSH DNS
+      - task: update-runtime-config
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: pivotalcfreleng/bbl
+          inputs:
+            - name: bbl-state
+            - name: stackdriver-tools-source
+          run:
+            dir: bbl-state
+            path: sh
+            args: [-c, "eval \"$(bbl print-env)\" && bosh --non-interactive update-runtime-config ../stackdriver-tools-source/ci/runtime-config.yml"]
+      # Gets the CF deployment source (manifest/releases/operations)
+      - get: cf-deployment-source
+      # Gets the latest ubuntu stemcell
+      - get: gcp-xenial-stemcells
+      # Patches the deployment manifest to use the stemcell provided by the previous step.
+      - task: update-manifest
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: mikefarah/yq
+          inputs:
+            - name: cf-deployment-source
+            - name: gcp-xenial-stemcells
+          outputs:
+            - name: edited-manifest
+          run:
+            path: sh
+            args: ["-c", "mkdir -p edited-manifest && yq w cf-deployment-source/cf-deployment.yml stemcells[0].version \"$(cat gcp-xenial-stemcells/version)\" > edited-manifest/cf-deployment.yml"]
+      # Performs a BOSH deploy to deploy Cloud Foundry on the BOSH director
+      - put: cf
+        params:
+          source_file: bbl-state/bdr-source-file
+          manifest: edited-manifest/cf-deployment.yml
+          stemcells:
+            - gcp-xenial-stemcells/stemcell.tgz
+          vars:
+            enable_ssl: false
+            system_domain: ((domain_name))
+          ops_files:
+            - cf-deployment-source/operations/scale-to-one-az.yml
+            - cf-deployment-source/operations/use-compiled-releases.yml
+  # Stackdriver-tools CI/CD
+  - name: unit-test
+    plan:
+      - get: stackdriver-tools-source
+        attempts: 2
+        trigger: true
+      - task: unit-test
```

Comment:
I notice that all the tasks have been inlined in the pipeline rather than referring to external files. It might make this file easier to read if it were split apart.